src/core/component.js

```diff
@@ -136,6 +126,38 @@ Component.prototype = {
     return utils.extend({}, data);
   },
 
+  updateCachedValue: function (value, isSinglePropSchema) {
```

Comment:
We call it `cachedValue` in one place and `attrValue` in another. Should we call it `cachedAttrValue` everywhere?